android/capacitor/src/main/java/com/getcapacitor/WebViewLocalServer.java

```diff
@@ -172,7 +172,7 @@ public WebResourceResponse shouldInterceptRequest(WebResourceRequest request) {
       return null;
     }
 
-    if (isLocalFile(loadingUrl) || (bridge.getConfig().getString("server.url") == null && !bridge.getAppAllowNavigationMask().matches(loadingUrl.getHost()))) {
+    if (isLocalFile(loadingUrl) || loadingUrl.getHost().contains(bridge.getHost()) || (bridge.getServerUrl() == null && !bridge.getAppAllowNavigationMask().matches(loadingUrl.getHost()))) {
```

Comment:
Would this incorrectly trigger if `loadingUrl.getHost() = "something.app"` and `bridge.getHost() = "app"`?

Reply:
changed the `contains` to `equalsIgnoreCase`